src/actions/trigger-action/index.ts

```diff
@@ -14,6 +14,7 @@ interface Context {
   payload: {
     inputs: {
       email: string
+      repository: string
```

Comment:
None of the other calls to trigger action (`approvePullRequest`, `createRelease` etc) will pass this param, so it should be optional if we decide to keep it (although see comment below for further discussion on this param).

```suggestion
      repository?: string
```